src/unix/system_structs.h

```diff
@@ -83,6 +83,7 @@ typedef struct iovec {
 #define EMSGSIZE        90		/* Message too long */
 #define EPROTOTYPE      91		/* Wrong protocol type for socket */
 #define EOPNOTSUPP      95		/* Operation not supported */
+#define ECONNABORTED    103     /* Software caused connection abort */
```

Comment:
Nit: alignment (I know, the presence of tabs in the preceding lines doesn't help)